userday2021/frasil_EDF_OTUsers2021/.gitignore

```diff
@@ -0,0 +1,9 @@
+*.aux
+*.bbl
+*.blg
+*.out
+*.log
+*.toc
+*.idx
+*.snm
+*.nav

```

Comment:
The root gitignore should already have everything

Reply:
This is true. Fixed.